README.md

```diff
@@ -18,7 +18,7 @@ Work on GFO has started in 1999 in the context of the GOL project (General Ontol
 The "native" formalization language for GFO is [first-order logic (FOL)](https://en.wikipedia.org/wiki/First-order_logic). Partial axiomatizations of GFO in FOL exist in report working drafts, but are not yet available to the public.
 
 ### OWL
-There is an [OWL](http://en.wikipedia.org/wiki/Web_Ontology_Language) version of GFO, which currently comprises a stable core called *gfo-basic.owl* and a more extensive version called *gfo.owl*.
+There is an [OWL](http://en.wikipedia.org/wiki/Web_Ontology_Language) version of GFO, which currently comprises a stable core called *gfo-basic.owl* and a more extensive version called *gfo.owl*. *gfo-light.owl* contains basic concepts and axioms described in the current GFO papers and is primarily intended to found domain or application ontologies.
```

Comment:
Why keeping the reference to gfo-basic.owl at this point? Isn't gfo-light.owl meant to replace it?

Reply:
I have described the distinction between gfo-basic and gfo-light in the README.